src/Microsoft.AspNet.OData.Shared/Builder/EdmModelHelperMethods.cs

```diff
@@ -440,6 +440,13 @@ private static void ValidateActionOverload(IEnumerable<ActionConfiguration> conf
             // add annotation for properties
             Dictionary<PropertyInfo, IEdmProperty> edmProperties = edmTypeMap.EdmProperties;
             model.AddClrPropertyInfoAnnotations(edmProperties);
+
+            // add Enum mapping annotation
+            if (edmTypeMap.EnumMembers != null && edmTypeMap.EnumMembers.Any())
+            {
+                model.SetAnnotationValue(model, new ClrEnumMemberAnnotation(edmTypeMap.EnumMembers));
```

Comment:
>model [](start = 41, length = 5)

So we are adding the enum annotations to the model as top level, instead of associating them with the actual Enum type in which they are defined? Annotations for a specific enum type should have a local scope instead of global (model-wide) scope, right?